data/mods/Aftershock/player/bionics.json

```
@@ -83,6 +83,38 @@
     "react_cost": "12 kJ",
     "time": "1 s"
   },
+  {
+    "id": "afs_bio_blood_plus",
+    "type": "bionic",
+    "name": { "str": "Synthetic Marrow" },
+    "description": "The interior of your femurs have been scrapped clean, the marrow replaced with a Mercurial brand bio-factory that produces high efficiency blood analogues.  The blood substitutes are fine-tuned for combat, increasing stamina, disease resistance and clotting speed.",
+    "cant_remove_reason": "The biofactory has perfectly integrated with your body, removing it would be pointless and potentially fatal.",
+    "occupied_bodyparts": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "enchantments": [
+      {
+        "ench_effects": [ { "effect": "synth_blood_permanent", "intensity": 2 } ],
+        "values": [ { "value": "CARDIO_MULTIPLIER", "multiply": 1.2 } ]
```

Multiplier 1.2 means 2.2x increase , that's how enchantment math works 
```suggestion
        "values": [ { "value": "CARDIO_MULTIPLIER", "multiply": 0.2 } ]
```

---

Oh no this is chromed up warmachine-psycho tier stuff, its meant to be a duplication of stamina.